.github/workflows/npm-publish.yml

```diff
@@ -19,31 +19,30 @@ jobs:
         with:
           ref: ${{ github.event.inputs.tag }}
 
-      - name: Match semver pattern
-        uses: actions-ecosystem/action-regex-match@v2
-        id: regex-match
-        with:
-          text: ${{ github.event.inputs.tag }}
-          regex: '^v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$'
-
       - name: Check package version
         id: cpv
         uses: PostHog/check-package-version@v2
 
+      - run: npm install semver
+
       - name: Validate tag
         uses: actions/github-script@v6
         with:
           script: |
-            const match = `${{ steps.regex-match.outputs.match }}`;
-            if(match === '') {
-              core.setFailed(`Tag ${context.payload.inputs.tag} does not match semver pattern`);
-            } else {
+            const semver = require('semver')
+
+            const tag = `${context.payload.inputs.tag}`;
+            const version = tag.substring(1);
```

Comment:
`npx semver v1.2.3` also trims off the `v` for you

Reply:
`allowed-endpoints` in harden-runner does not support comments. I have created an [issue](https://github.com/step-security/harden-runner/issues/199) to track it. Removed `hooks.slack.com:443` for now.